libraries/botbuilder/src/activityValidator.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Activity } from 'botbuilder-core';
+
+export function validateActivity(resolve: any, activity: Activity): void{
+    if (typeof activity !== 'object') { throw new Error(`activityValidator.validateActivity(): invalid request body.`); }
+    if (typeof activity.type !== 'string') { throw new Error(`activityValidator.validateActivity(): missing activity type.`); }
```

Comment:
I would change the error message to use `validateActivity()` instead of `activityvalidator.validateActivity()`.